src/pages/Profile/subpages/Settings/Contact.js

```diff
@@ -62,20 +72,20 @@ export default function Contact({ social, email, phoneNumber }) {
           </span>
           {!edit ? (
             <span className="contact-social-input contact-data">
-              {currentInstagram}
+              {userSocials[0] !== undefined ? userSocials[0].link : ""}
             </span>
           ) : (
             <input
               className="contact-social-input"
-              onChange={e => handleInstagramChange(e.target.value)}
-              value={currentInstagram}
+              onChange={e => handleSocialHelper(0, e.target.value)}
+              defaultValue = {userSocials[0] !== undefined ? userSocials[0].link : ""}
             ></input>
```

Comment:
It's hard to tell whether the inputs are editable or not. Instead of showing inputs when in 'non-editable' we could just show text and when the user clicks `edit` we can change it to inputs.
We should also specify whether the user just needs to put the handle or the full url. Maybe next to the Social header we can say `input the full url`.